packages/vite/src/node/server/middlewares/error.ts

```diff
@@ -15,7 +15,7 @@ export function prepareError(err: Error | RollupError): ErrorPayload['err'] {
     id: (err as RollupError).id,
     frame: strip((err as RollupError).frame || ''),
     plugin: (err as RollupError).plugin,
-    pluginCode: (err as RollupError).pluginCode,
+    pluginCode: (err as RollupError).pluginCode?.toString(),
```

Comment:
Not sure whether we should deal with `object`-type `pluginCode` and `code`.

Reply:
I think that's fine for now 👍